src/main/java/ipman/IpMan.java

```diff
@@ -0,0 +1,41 @@
+package ipman;
+
+import ipman.commands.Command;
```

Comment:
I think it's good that you  put the import statements in lexicographical order which is a common style error that people make!